edgeml_pytorch/graph/rnn.py

```diff
@@ -70,16 +70,16 @@ def __init__(self, input_size, hidden_size,
         self._hidden_size = hidden_size
         self._gate_nonlinearity = gate_nonlinearity
         self._update_nonlinearity = update_nonlinearity
-        #self._num_weight_matrices = num_weight_matrices
+        #self._num_weight_matrices = [1,1]
```

Comment:
Is it better to set it to None, as opposed to [1,1]? @adityakusupati

Reply:
@harsha-simhadri  that line has been commented out, but it should be None.

Reply:
We can remove it also, as `self._num_weight_matrices` is assigned here https://github.com/pushkalkatara/EdgeML/blob/8a1189065b09a8c84abebb8e2250a7c3c2bb571d/edgeml_pytorch/graph/rnn.py#L228

Reply:
@pushkalkatara should be part of the base RNN class hence we need to assign None here.

Reply:
Okay, I'll commit the change.

Reply:
@pushkalkatara I made some suggestion in a new review please do those change and I will do a comprehensive review today.